tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -459,7 +511,9 @@ can_sorted_merge_append(PlannerInfo *root, CompressionInfo *info, Chunk *chunk)
 
 	/* Ensure that we have path keys and the chunk is ordered */
 	if (pathkeys == NIL || ts_chunk_is_unordered(chunk))
+	{
```

Comment:
could you remove these brace changes from the PR. This makes rebasing unnecessary harder.

Reply:
Yeah, that's a problem, I'll remove them. Added for debugging.